docs/en/SAMPLE_SKILLS.rst

```diff
@@ -86,17 +86,27 @@ Demonstrates how to prompt and parse multiple values from customers using
 and `Entity Resolution <https://developer.amazon.com/docs/custom-skills/define-synonyms-and-ids-for-slot-type-values-entity-resolution.html>`_.
 
 `High Low Game <https://github.com/alexa/skill-sample-python-highlowgame>`_
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Template for a basic high-low game skill. When the user guesses a number,
 Alexa tells the user whether the number she has in mind is higher or lower.
 Demonstrates use of persistence attributes and the persistence adapter
 in the SDK.
 
 `AudioPlayer SingleStream and MultiStream <https://github.com/alexa/skill-sample-python-audio-player>`_
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Sample skills that show how to use `AudioPlayer interface <https://developer.amazon.com/public/solutions/alexa/alexa-skills-kit/docs/custom-audioplayer-interface-reference>`__ and `PlaybackController interface <https://developer.amazon.com/public/solutions/alexa/alexa-skills-kit/docs/custom-playbackcontroller-interface-reference>`__
 in Alexa, to build audioplayer skills. The SingleStream skill sample demonstrates how to create a live radio skill, 
 along with localization support. The MultiStream skill sample demonstrates how to create 
-a basic podcast skill that can play multiple, pre-recorded audio streams. 
+a basic podcast skill that can play multiple, pre-recorded audio streams.
+
+`Pager Karaoke <https://github.com/alexa-labs/skill-sample-python-pager-karaoke>`_
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This sample demonstrates 3 features of APL: the
+`Pager Component <https://developer.amazon.com/docs/alexa-presentation-language/apl-pager.html>`__,
+`SpeakItem Command <https://developer.amazon.com/docs/alexa-presentation-language/apl-commands-components.html>`__,
```

Comment:
This link is showing a `page not found`. Has it been moved to some other url?

Reply:
Good catch. Looks like the link changed between their initial release and now. Fixed it in both `Readme` & `Sample Skills` pages.